subprojects/core/src/main/java/org/gradle/internal/exceptions/LocationAwareException.java

```diff
@@ -31,50 +29,20 @@
  */
 @UsedByScanPlugin
 public class LocationAwareException extends GradleException implements FailureResolutionAware {
```

Comment:
❌ We shouldn't change this exception, it is meant for failures that *do have* a source and line number.

Reply:
Based on [my comment below](https://github.com/gradle/gradle/pull/3135#discussion_r144067599) does this change your opinion?